doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-drd-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='dmn-gs-creating-drd-proc']
+= Creating the traffic violations DMN decision requirements diagram (DRD)
+
+Use the DMN designer in {CENTRAL} to design the DMN decision requirements diagrams (DRDs) and define decision logic for the traffic violation project.
+
+.DRD for the Traffic Violation example
+image::dmn/dmn-traffic-violations-drd.png[]
+
+.Prerequisite
+* You have created the traffic-violations project in {CENTRAL}.
+
+.Procedure
+. On the *traffic-violations* project's home screen, click *Add Asset* to open the *Add Asset* screen.
+. Click *DMN* to open the *Create new DMN* dialog.
+.. Enter `Traffic Violation` in the *Name* field.
+.. From the *Package* list, select `org.kie.example.traffic.traffic_violations`.
+.. Click *+Ok* to open the DMN asset in the DMN designer.
```

Comment:
Please remove the "+" from all "Ok" refs.

Reply:
Changed as suggested.